zfdg.csl

```diff
@@ -0,0 +1,384 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-model href="schema-master/csl.rnc" type="application/relax-ng-compact-syntax"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="de-DE">
+  <!--
+================================================================================
+INFORMATIONEN ZUM ZITIERSTIL
+================================================================================
+-->
+ <info>
+    <title>Zeitschrift für digitale Geisteswissenschaften</title>
+    <title-short>ZfdG</title-short>
+    <!-- hier zoterodaten ergänzen -->
+    <id>http://www.zotero.org/styles/zfdg</id>
+    <link href="http://www.zotero.org/styles/zfdg" rel="self"/>
+    <!-- URI of the style from which the current style is derived -->   
```

Comment:
Please add a template style that you used to make this style.

Reply:
Done.